1.5/Source/PawnStorages/PawnStorages/JobDriver_Release.cs

```diff
@@ -46,11 +46,17 @@ public override IEnumerable<Toil> MakeNewToils()
         {
             Pawn actor = release.actor;
             CompPawnStorage comp = TargetA.Thing.TryGetComp<CompPawnStorage>();
+            // Not sure why this is needed for animals and entities, but it shouldn't do any harm
+            CompAssignableToPawn_PawnStorage compAssignableToPawn = TargetA.Thing.TryGetComp<CompAssignableToPawn_PawnStorage>();
```

Comment:
you're not using this comp anymore so this can go

Reply:
Removed